paddle/fluid/pybind/eager_functions.cc

```diff
@@ -523,8 +523,8 @@ static PyObject* eager_api_sparse_coo_tensor(PyObject* self,
       std::dynamic_pointer_cast<phi::DenseTensor>(non_zero_indices.impl());
   auto dense_elements =
       std::dynamic_pointer_cast<phi::DenseTensor>(non_zero_elements.impl());
-  // TODO(zhangkaihuo): After create SparseTensor, call coalesced() to sort and
-  // merge duplicate indices
+  // TODO(zhangkaihuo): After creating SparseCooTensor, call coalesced() to sort
+  // and merge duplicate indices
```

Comment:
For now, coalesced() is just used to determine if  the sparse coo tensor has been coalesced.  But  there is no corresponding implementation of how to coalesce.  Maybe we can  add  two methods:  coalesce() to actually  coalesce,  is_coalesced()  to  determine if the tensor has been coalesced.